### DISCUSSION TITLE
chinese-internet-is-accelerating/

### DISCUSSION THREAD

**Discussion by giscus** [eallion/eallion.com#134] [Giscus]
# chinese-internet-is-accelerating/

AI Summary Pre-generated by ChatGPT 3.5

https://www.eallion.com/chinese-internet-is-accelerating/

**Comment by rebron1900** [comment]
倒不至于说完全消失不见了，只是原有途径可能不存在了，但是会碎片化的存在于互联网上，直到最后一个人放弃保留这些资料之时才是真正消失，不要太悲观

**Reply by eallion** [reply]
但是这样的说法感觉像是自我安慰。
